pkg/test/framework/features/features.yaml

```diff
@@ -16,26 +16,26 @@
 #"values" is a keyword and must contains a list of strings. "values" is omitted when building the label name
 #   This is used when a map needs direct leaf nodes but also other maps
 features:
-  values: 
-    - observability
+  observability:
   traffic:
     circuit-breakers:
-        - fail-gracefully
+        fail-gracefully:
```

Comment:
is the value a string or `{}` ?

Reply:
The value of a leaf node is nil.  The colon is necessary to be valid yaml, and to make it easy to use an old leaf node as a new branch node.

Reply:
Thanks please add this in the inline comments, otherwise lgtm